src/py/flwr/server/strategy/fedavg.py

```diff
@@ -211,16 +211,17 @@ def configure_evaluate(
         self, rnd: int, parameters: Parameters, client_manager: ClientManager
     ) -> List[Tuple[ClientProxy, EvaluateIns]]:
         """Configure the next round of evaluation."""
-        # Do not configure federated evaluation if a centralized evaluation
-        # function is provided
-        if self.eval_fn is not None:
```

Comment:
We could simply change this line to check if `fraction_eval` is `0.0` (and return an empty list if it is). That'd allow both distributed and federated evaluation, but it'd enable users to disable federated evaluation by simply setting `fraction_eval` to `0.0`.